plugins/headlamp-plugin/test-headlamp-plugin.sh

```diff
@@ -53,3 +53,22 @@ npm run lint-fix
 
 # test type script error checks
 npm run tsc
+
+# test upgrade adds missing files
+rm jsconfig.json tsconfig.json src/headlamp-plugin.d.ts
+node ../bin/headlamp-plugin.js upgrade --skip-package-updates
+stat jsconfig.json
+stat tsconfig.json
+stat src/headlamp-plugin.d.ts
+
+# change @kinvolk/headlamp-plugin version in package.json to an old one "^0.4.9"
+python3 -c "import sys;sys.stdout.write(\"\".join([l if '\"@kinvolk/headlamp-plugin\"' not in l else '    \"@kinvolk/headlamp-plugin\": \"^0.4.9\"\n' for l in open(\"package.json\", \"r\").readlines()]))" > /tmp/package.json.bak && mv /tmp/package.json.bak package.json
+
+# test upgrade updates the package line, and the old version is not in there
+node ../bin/headlamp-plugin.js upgrade
+echo "
+import sys
+old_version = '0.4.9'
+if old_version in open('package.json').read():
+  sys.exit(1)
+" | python3
```

Comment:
I was wondering why this is not using just sed/grep and other shell tools, but remembered we need this to run also in Windows without installing a bunch of tools.
Will we have Python in the Windows CI though? I wonder if this should just be js since we have node already.

Reply:
Yes, I'll fix the tests to use nodejs.

Reply:
(Also so the tests can be run on windows cmd like we spoke about another time)

Reply:
Converting tests to js was done in the other PR. This one now uses JS for these new tests instead of bash+python.